janus_core/cli/geomopt.py

```diff
@@ -42,6 +97,10 @@ def geomopt(
     # numpydoc ignore=PR02
     ctx: Context,
     struct: StructPath,
+    optimizer: Annotated[
+        str,
+        Option(help="Name of ASE optimizer function to use.  [default: LBFGS]"),
```

Comment:
Different form of default to other things?

Reply:
Ah yes, I changed my mind about relying on geom_opt defaults. Fixed